frontend/package-lock.json

```diff
@@ -21,6 +21,7 @@
         "clsx": "^2.1.0",
         "cmdk": "^0.2.1",
         "framer-motion": "^11.0.24",
+        "googleapis": "^144.0.0",
         "lucide-react": "^0.453.0",
```

Comment:
please rmove this file from the comiit and also the frontend.yarn.lock file. They would be autogenerated

Reply:
@Dprof-in-tech  Thanks for the quick reply, did it